website/content/docs/v0.3/Guides/bootstrapping.md

```diff
@@ -178,14 +202,15 @@ For instance:
 ```bash
 export CONTROL_PLANE_SERVERCLASS=any
 export WORKER_SERVERCLASS=any
-export KUBERNETES_VERSION=v1.20.1
+export TALOS_VERSION=v0.11.5
+export KUBERNETES_VERSION=v1.21.3
```

Comment:
```suggestion
export KUBERNETES_VERSION=v1.21.3
```

Reply:
This is kind of an issue, because the `TALOS_VERSION` is required. Without it the `clusterctl config cluster management-plane -i sidero` throws an error and does nothing.